app/src/main/res/layout/activity_add_playlist.xml

```diff
@@ -2,6 +2,7 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/AddPlaylistRootView"
```

Comment:
I can't find where you use this id again. Is it used implicitly? If not, I think you can remove it as it may prevent you from having some merge conflicts. ;)

Reply:
I am using it in an onTouchListener to cancel the deletion if the user clicks anywhere except the snackbar (unless he clicks one the the layout's elements but this is a detail)

Reply:
> unless he clicks one the the layout's elements

I didn't see that. I don't think this is really intuitive either. The snackbar should be the only place where you could confirm (or undo) the deletion in my opinion.

Reply:
It is a little bit hard to imagine just when reading the code but it makes sense if you use it. Basically the snackbar is on a timer and if you don't click delete in time it will stop the deletion process. Adding touch listener's means one can cancel the deletion process more quickly by clicking anywhere else on the screen. Try it and tell me what you think